DESCRIPTION.md

```diff
@@ -11,6 +11,7 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 - v3.12.4(TBD)
```

Comment:
Should this still be TBD?

Reply:
Unless you can tell me when the next release is happening yes

Reply:
🧞